templates/motd.erb

```diff
@@ -1,5 +1,6 @@
-The operating system is <%= @operatingsystem %>
-The free memory is <%= @memoryfree %>
-<%- if @domain -%>
-The domain is <%= @domain %>
-<%- end -%>
+<%= @operatingsystem %> <%= @operatingsystemrelease %> <%= @architecture %>
+
+FQDN:         <%= @fqdn %> (<%= @ipaddress %>)
+Processor:    <%= @processor1 %>
```

Comment:
This should probably be `processor0` or `processor1`.  On a single physical CPU with multiple cores, only processor0 exists.